design-editor/src/pane/guide.js

```diff
@@ -105,6 +107,18 @@ class Guide {
 				rule = this._getInsertedRule(targetInfo, containerInfo.$element, isRestrict, containerInfo);
 			}
 
+			/**
+			 * This should be fixed during refactor
+			 * Header as a element had parent-constraint but cannot be append to the end of siblings elements
+			 * like it is coded in 93. line
+			 */
```

Comment:
It looks like a TODO, so ... it'd be more readable to add @todo

Reply:
Done